src/vision/VisionProcessor.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef VISION_PROCESSOR_H
+#define VISION_PROCESSOR_H
+#include <ros/ros.h>
```

Comment:
I will remove it from Vision.cpp, didn't notice the redundancy. This is proper according to [this question](http://stackoverflow.com/questions/21593/what-is-the-difference-between-include-filename-and-include-filename).